config/deploy.rb

```diff
@@ -86,10 +86,17 @@
   desc 'Modify permissions on /srv/idp'
   task :mod_perms do
     on roles(:web), in: :parallel do
-      execute :sudo, :chown, '-R', 'ubuntu:nogroup', deploy_to
+      execute :sudo, :chown, '-R', 'ubuntu:nogroup', deploy_to + '/current', '/srv/idp/shared'
```

Comment:
ACTUALLY PETER, I'm going to cut you off right there 🎙 ....

Peter doesn't always test his PRs, but when he does, he does it while promoting code to production.